### PR TITLE
doc: revise SubmittingPatches

### PR DESCRIPTION
**Pull request by tchaikov**
* rename SubmittingPatches to SubmittingPatches.rst for better formatting
* add doc for the prefix in the commit title
* add doc regarding to "Fixes:" tag

### CODE REVIEW

File: SubmittingPatches.rst [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+      You don't. If someone reviews your pull request, they should indicate they
+      have done so by commenting on it with "+1", "looks good to me", "LGTM",
+      and/or the entire "Reviewed-by: ..." line with their name and email address.
+
+      The developer merging the pull request should note positive reviews and
+      include the appropriate Reviewed-by: lines in the merge commit.
+
+2. Patch submission via ceph-devel@vger.kernel.org
+
+   The best way to generate a patch for manual submission is to work from
```
Comment (theanalyst) [inline SubmittingPatches.rst]: Should we add a small note here saying for the kernel client we want patches in mailing list and for everything else it is preferred to submit in Github for review?

Comment (theanalyst) [inline SubmittingPatches.rst]: Probably we can add the note at the top of this para .. ie . above github Pull request?

Comment (tchaikov) [inline SubmittingPatches.rst]: @ukernel what do you think about @theanalyst 's suggestion? i don't have experience in kernel space, so not quite about the our convention of submitting kernel client patches. but i'd see traffic of kernel patches in ceph-devel@vger.kernel.org. so i think @theanalyst 's comment makes sense. just want get your bless for adding this note.

**Comment by tchaikov** [comment]
@theanalyst updated to address your comments.

**Comment by theanalyst** [comment]
LGTM

**Pull request merged**